portfolio/src/main/webapp/style.css

```diff
@@ -156,4 +156,5 @@
 
   .sea-img {
     height: 198px;
-  }
\ No newline at end of file
+  }
+  
```

Comment:
there a newline issue here

Reply:
Hi Kevin, did I understand wrongly? I thought it was a convention to put a newline at the end of the file

Reply:
I think there are characters on the empty line, normally github will not show the (-) icon

Reply:
can you check if there's any spaces on the empty line?

Reply:
Yup there was, thank you! Removed it.